docs/frontend_development.md

```diff
@@ -84,3 +84,17 @@ $ hass --skip-pip
 ```
 
 [hass-polymer]: https://github.com/home-assistant/home-assistant-polymer
+
+## Troubleshooting
+
+If you've set up everything according to the documentation, but it still looks like your frontend isn't using the correct source, you might need to check and change a few things:
+
+> Instructions are for Google Chrome
+
+1. Disable cache by ticking the box in `Network` > `Disable cache`
+
+<img width="367" alt="disable-cache" src="https://user-images.githubusercontent.com/7738048/42022916-cfb9bc8c-7abe-11e8-82d3-ca125c949766.png">
+
+2. Enable Bypass for network in `Application` > `Service Workers` > `Bypass for network` or `Unregister` the service worker
```

Comment:
Unregister the service worker will unregister a bad one, however if that's the current one it will be registered right away. Better to not mention it and just mention bypass